_includes/masthead.html

```diff
@@ -21,9 +21,13 @@
         {% if site.search == true %}
         <button class="search__toggle" type="button">
           <span class="visually-hidden">{{ site.data.ui-text[site.locale].search_label | default: "Toggle search" }}</span>
+          {% if site.search_icon and site.search_icon != 'builtin' %}
+          <i class="{{ site.search_icon }}"></i>
```

Comment:
Can you fix the indentation (add a level) for this line and the 3 lines under (the `<svg>` element)?